README-minikube.md

```diff
@@ -0,0 +1,234 @@
+    ssl-redirect: "false"
+  scope:
+    enabled: true
+    namespace: $DESIREDNAMESPACE
+EOF
+
+helm install stable/nginx-ingress --version=0.12.3 -f ingressvalues.yaml \
+--namespace $DESIREDNAMESPACE
+```
+
+
+** !Optional **
+
+If you want your own certificate set here you should create a secret from your cert files:
+
+```bash
+kubectl create secret tls certsecret --key /tmp/tls.key --cert /tmp/tls.crt --namespace $DESIREDNAMESPACE
+```
+
+Then deploy the ingress with following settings
+```bash
+cat <<EOF > ingressvalues.yaml
+controller:
+  config:
+    ssl-redirect: "false"
+  scope:
+    enabled: true
+    namespace: $DESIREDNAMESPACE
+  publishService:
+    enabled: true
+  extraArgs:
+    default-ssl-certificate: $DESIREDNAMESPACE/certsecret
+EOF
+
+helm install stable/nginx-ingress --version=0.12.3 -f ingressvalues.yaml \
+--namespace $DESIREDNAMESPACE
+```
+
+
+### 2. Get the nginx-ingress-controller release name from the previous command and set it as a varible:
+```bash
+export INGRESSRELEASE=knobby-wolf
+```
+
+### 3. Wait for the nginx-ingress-controller release to get deployed:
+```bash
+helm status $INGRESSRELEASE
+```
+
+*Note:* When checking status, your pods should be ```READY 1/1```
+
+### 4. Get the nginx-ingress-controller port for the infrastructure :
+```bash
+export INFRAPORT=$(kubectl get service $INGRESSRELEASE-nginx-ingress-controller --namespace $DESIREDNAMESPACE -o jsonpath={.spec.ports[0].nodePort})
+```
+
+### 5. Get Minikube or ELB IP and set it as a variable for future use:
+
+```bash
+export ELBADDRESS=$(minikube ip)
+```
+
+### 6. Add the Alfresco Kubernetes repository to helm.
+
+```helm repo add alfresco-incubator http://kubernetes-charts.alfresco.com/incubator```
+
+### 7. Deploy the DBP
+
+```bash
+helm install alfresco-incubator/alfresco-dbp \
+--set alfresco-infrastructure.alfresco-api-gateway.keycloakURL="http://$ELBADDRESS:$INFRAPORT/auth/" \
+--set alfresco-infrastructure.alfresco-identity-service.ingressHostName="$ELBADDRESS:$INFRAPORT" \
+--set alfresco-content-services.repository.environment.IDENTITY_SERVICE_URI="http://$ELBADDRESS:$INFRAPORT/auth" \
+--namespace=$DESIREDNAMESPACE
+```
+
+You can either deploy the dbp fully or choose the components you need for your specific case. 
+By default the dbp chart will deploy fully.
+
+To disable specific components you can set the following values to false when deploying:
+```
+alfresco-content-services.enabled
+alfresco-process-services.enabled
+alfresco-sync-service.enabled
+```
+
+Example: For disabling sync-service you will need to append the following subcommand to the helm install command:
+```bash
+ --set alfresco-sync-service.enabled=false 
+```
+
+### 8. Get the DBP release name from the previous command and set it as a variable:
+
+```bash
+export DBPRELEASE=littering-lizzard
+```
+
+### 9. Checkout the status of your DBP deployment:
+
+```bash
+helm status $INGRESSRELEASE
+helm status $DBPRELEASE
+
+# Open Alfresco Repository in Browser
```

Comment:
I think our NOTES template spits these URLs out after deploying.  Should we just reference that here rather than listing out commands? (I don't have a strong opinion on it, just a potential area for simplification).

Reply:
I concur. I'd like to and attach a sample minikube output to easier illustrate this point, however; at the moment I am unable to deploy the whole of DBP due to hardware constraints of the environments at my disposal.